src/rpc/cli.cpp

```diff
@@ -210,7 +210,7 @@ static int cli_completion(char *token, char ***array)
  */
 static int cli_check_secret(const char *source)
 {
-   if (boost::regex_match(source, cli_regex_secret()))
+   if (nullptr != cli_regex_secret().expression() && boost::regex_match(source, cli_regex_secret()))
```

Comment:
Please use https://www.boost.org/doc/libs/1_56_0/libs/regex/doc/html/boost_regex/ref/basic_regex.html#boost_regex.basic_regex.empty for checking.

Reply:
Ok sure ! Thanks !

Reply:
Done